Library/Homebrew/software_spec.rb

```diff
@@ -504,14 +504,19 @@ def root_url(var = nil, specs = {})
     end
   end
 
-  sig { params(tag: Utils::Bottles::Tag).returns(T::Boolean) }
-  def compatible_locations?(tag: Utils::Bottles.tag)
+  sig { params(tag: Utils::Bottles::Tag).returns(String) }
+  def tag_to_cellar(tag: Utils::Bottles.tag)
```

Comment:
One small stylistic question: is it preferable to make `tag` a named parameter here?  Doing so does require us to use `tag_to_cellar(tag: TAG)` which seems preferable to me but is slightly more verbose than just `tag_to_cellar(TAG)` if a named parameter was not used.

Reply:
I don't think it's preferable unless it's not obvious what the parameter is. `tag_to_cellar` with a single parameter implies to me that `tag` would be that parameter.

```suggestion
  def tag_to_cellar(tag = Utils::Bottles.tag)
```

Reply:
That makes sense.  I've changed this accordingly.